examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3311,6 +3311,7 @@ class GLTFParser {
 			// baseColorTexture, emissiveTexture, and specularGlossinessTexture use sRGB encoding.
 			if ( material.map ) material.map.encoding = sRGBEncoding;
 			if ( material.emissiveMap ) material.emissiveMap.encoding = sRGBEncoding;
+			if ( material.specularMap ) material.specularMap.encoding = sRGBEncoding;
```

Comment:
Let's add the remaining textures from `KHR_materials_specular` and `KHR_materials_sheen` as well —

```suggestion
			if ( material.specularMap ) material.specularMap.encoding = sRGBEncoding;
			if ( material.specularColorMap ) material.specularColorMap.encoding = sRGBEncoding;
			if ( material.sheenColorMap ) material.sheenColorMap.encoding = sRGBEncoding;
```